contracts/Remittance.sol

```diff
@@ -1,16 +1,140 @@
 //SPDX-License-Identifier: MIT
 pragma solidity >= 0.6.0 < 0.7.0;
 
-import "../node_modules/@openzeppelin/contracts/access/Ownable.sol";
-import "../node_modules/@openzeppelin/contracts/utils/Pausable.sol";
+import "./access/Ownable.sol";
+import "./utils/Pausable.sol";
 
-import {SafeMath} from "../node_modules/@openzeppelin/contracts/math/SafeMath.sol";
+import {SafeMath} from "./math/SafeMath.sol";
 
 contract Remittance is Ownable, Pausable {
 
-    using SafeMath for uint;
+    using SafeMath for uint256;
 
+    mapping(bytes32 => RemittanceInstance) public remittances;
 
+    struct RemittanceInstance {
+        address funder;
+        uint256 fundsOwed;
+        uint256 expiryDate;
+    }
+
+    event RemittanceFundsCreated(
+        bytes32 indexed hash,
+        bytes32 passwordBrokerHash,
+        address indexed funder,
+        address indexed broker,
+        uint256 amount,
+        uint256 expiryDate
+    );
+
+    event RemittanceFundsReleased(
+        bytes32 indexed hash,
+        bytes32 indexed password,
+        address indexed broker,
+        uint256 amount
+    );
+
+    event RemittanceFundsReclaimed(
+        bytes32 indexed hash,
+        bytes32 indexed passwordBrokerHash,
+        address indexed funder,
+        uint256 amount
+    );
+
+    // The funder can create a remittance
+    // Hash is a sha256 hash of the broker address + recipient password
+    function create(bytes32 passwordBrokerHash, address broker, uint256 expiryDate)
+    external
+    payable
+    whenNotPaused
+    returns(bool success)
+    {
+        require(broker != msg.sender, "Caller cannot be the broker");
+        require(msg.value > 0, "Amount must be greater than 0");
+        require(broker != address(0), "Address cannot be zero");
+
+        // A 15 min margin has been added to allow for any lag
+        require(expiryDate >= block.timestamp + (24 * 60 * 60) - 1500, "Expiry less than 24h ahead");
+        checkEmptyHash(passwordBrokerHash);
+
+        bytes32 rehash = keccak256(abi.encodePacked(passwordBrokerHash, address(this)));
+
+        RemittanceInstance storage remittanceInstance = remittances[rehash];
+
+        // .funder must be empty otherwise the key is already in use
+        require(remittanceInstance.funder == address(0), "Remittance already exists");
+
+        remittanceInstance.funder = msg.sender;
+        remittanceInstance.fundsOwed = msg.value;
+        remittanceInstance.expiryDate = expiryDate;
+
+        emit RemittanceFundsCreated(rehash, passwordBrokerHash, msg.sender, broker, msg.value, expiryDate);
+
+        return true;
+    }
+
+    // The broker sends the recipient's password to release the funds
+    function release(bytes32 password) external whenNotPaused returns(bool success) {
+
+        bytes32 passwordBrokerHash = hashPasswordBroker(password, address(msg.sender));
+        bytes32 rehash = keccak256(abi.encodePacked(passwordBrokerHash, address(this)));
```

Comment:
You are using a formula here and another one, of wait the same one actually, on line 60. May they always be updated together, forever in the future ;).